File: Implementation/electronics-shop/electronics-shop.js
function getMoneySpent(keyboards, drives, s) {
  highestCost = 0;
  for (let i = 0; i < keyboards.length; i++) {
    for (let j = 0; j < drives.length; j++) {
      let cost = keyboards[i] + drives[j];
      if (cost <= s) {
        if (cost > highestCost) {
          highestCost = cost;
        }
      }
    }
  }
  if (highestCost > 0) {
    return highestCost;
  } else return -1;
}

File: Implementation/electronics-shop/electronics-shop.py
def getMoneySpent(keyboards, drives, s):
    costArray = []
    for i in range(len(keyboards)):
        for j in range(len(drives)):
            cost = keyboards[i] + drives[j]
            if cost < s:
                costArray.append(cost)
    if len(costArray) > 0:
        print('Out', max(costArray))
        return max(costArray)
    else:
        return -1
